grassroots-backend/src/phone-canvass/PhoneCanvass.service.ts

```diff
@@ -33,16 +40,22 @@ export class PhoneCanvassService {
     });
     await this.entityManager.flush();
 
-    canvass.contacts.forEach((canvasContact) => {
+    for (const canvasContact of canvass.contacts) {
+      let contact: RequiredEntityData<ContactEntity> | null =
+        await this.entityManager.findOne(ContactEntity, {
+          gvote_id: canvasContact.contact.gvote_id,
```

Comment:
I'm not sure if this is a good assumption for us right now. gvote contacts have multiple phone numbers and might have a different phone number on the fed and provincial side. I don't believe we have a concept of one contact having multiple phone numbers yet. If we do it this way, the contact will only ever have the phone number from the first time they're uploaded.

I'm not sure that updating based on gvote_id will work either.

Would it make sense just to create duplicate records for this first iteration and worry about merging them in the future?